### PR TITLE
docs/finding-legistar-id

### PR DESCRIPTION
**Pull request by dphoria**
### Link to Relevant Issue

https://github.com/CouncilDataProject/cdp-scrapers/issues/12

### Description of Changes

`finding_legistar_id.md`

### Question ###

Not sure where would be the best place to reference/link to this document. Proposing `event_gather_pipeline.md`.

### CODE REVIEW

**Review by sarahjliu — APPROVED** [review]
yay looks great! 

as for where this doc will be referenced, the instance creation issue template would have a link to it and we can also add the doc to https://councildataproject.org/cdp-scrapers/. The event gather readme could be another place we surface this but @JacksonMaxfield prob has a more holistic understanding of the documentation structure and where this one could go

**Comment by dphoria** [comment]
> 
> 
> yay looks great!
> 
> as for where this doc will be referenced, the instance creation issue template would have a link to it and we can also add the doc to https://councildataproject.org/cdp-scrapers/. The event gather readme could be another place we surface this but @JacksonMaxfield prob has a more holistic understanding of the documentation structure and where this one could go

OK we need an executive decision (LOL) from @JacksonMaxfield. This [finding_legistar_id.md](https://github.com/dphoria/cdp-backend/blob/main/docs/finding_legistar_id.md) should live in

1. `cdp-backend/docs` as in this PR.

2. `cdp-scrapers/docs` is Sarah' proposal (I think).

I... have no strong opinion either way as usual. :sweat_smile: This question also concerns/resolves [this](https://github.com/CouncilDataProject/cdp-scrapers/issues/12#issuecomment-899109344) comment. With that I'm going to snooze for the day haha. Thank you all for your help!

File: docs/finding_legistar_id.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Legistar is the most widely used legislation tracking tool in the nation. We are 
+leveraging its wide coverage to help standardize and streamline part of our onboarding 
+process by creating generalized Legistar scrapers so you can potentially skip the step 
+of creating an event scraper. Unfortunately a municipality’s Legistar ID isn’t always 
+easily discoverable so here are some steps to find it.
+
+1. Go to your municipality’s council page and locate the meeting agendas.
+
+2. Look at the URL of the website and if it contains “legistar”, then your 
+municipality uses Legistar. If it does not, your municipality does not use Legistar.
```
Comment (JacksonMaxfield) [inline docs/finding_legistar_id.md:12]: nit: I would change this to "likely does not use Legistar"

**Comment by JacksonMaxfield** [comment]
Ya sorry all. I think this is my bad / confusion / didn't know what already existed. I would place this in the cdp-scrapers repo. Option 2. `cdp-scrapers/docs`.

**Pull request closed without merging**